dev_environment/README.md

```diff
@@ -28,35 +31,69 @@ Here is a visual overview:
   `curl -X POST http://localhost:37080/l3af/configs/v1/update -d
   "@cfg/payload.json"`.  The `payload.json` file can be inspected and modified
   as desired. For more information on the L3AFD API see the [L3AFD API
   documentation](https://github.com/l3af-project/l3afd/tree/main/docs/api).
-* Verify the eBPF programs from `payload.json` are running by querying the
+9. Verify the eBPF programs from `payload.json` are running by querying the
   L3AFD debug API from the host: `curl http://localhost:38899/kfs/enp0s3`. This
   command assumes `enp0s3` is a valid network interface on the VM.
-* Once again send traffic to the VM web server:
+10. Once again send traffic to the VM web server:
   `hey -n 200 -c 20 http://localhost:18080`. The traffic will now be running
   through the eBPF programs (which may affect results dramatically depending
   on which eBPF programs are running and how they are configured).
-* To see the eBPF program metrics, browse to `http://localhost:33000` on the
+11. To see the eBPF program metrics, browse to `http://localhost:33000` on the
   host and login to Grafana with the default username and password of `admin`.
   After logging in you will be able to view the preconfigured dashboards.
+
+# Trying out L3AF on an Azure VM (Ubuntu 20.04.4 LTS)
+The following host prerequisites and installation instructions are for trying out L3AF on an Azure VM running an Ubuntu 20.04.4 LTS server.
+
+## Host Prerequisites
+
+* [L3AFD source code](https://github.com/l3af-project/l3afd)
+* [hey](https://github.com/rakyll/hey) or any HTTP load generator
+* A web browser
+* [Go for Linux](https://go.dev/doc/install) version 1.18.4 or greater
+
+## Installation Instructions
+1. Clone the [l3afd](https://github.com/l3af-project/l3afd.git) and [l3af-arch](https://github.com/l3af-project/l3af-arch.git)
+Github repositories to your Azure VM.
+2. Download and install Go for Linux version 1.18.4 or greater at https://go.dev/doc/install. Make sure that you do **not** use `sudo apt install`,
+because this command will install version 1.13 but the installation requires version 1.17 at the minimum.
+3. Edit `config.yaml` to point to the [L3AFD repository](https://github.com/l3af-project/l3afd) on your host machine.
+4. Run `install.sh` to install package dependencies, Grafana, Prometheus, the eBPF source code you need to build your eBPF programs against,
+and the eBPF package repository.
+5. On the Azure VM, go to your L3AFD directory and run `go install .`
+6. On the Azure VM, go to `~/go/bin` and run `l3afd` as root:
+  `sudo ./l3afd --config /vagrant/cfg/l3afd.cfg`.
+
+## Trying out your L3AFD Server
+1. Start test web servers by running the following commands on your Azure VM: `go run l3af-arch/dev_environment/code/web-server.go -port 8080 > /var/log/web-server1.log 2>&1 &` and `go run l3af-arch/dev_environment/code/web-server.go -port 8081 > /var/log/web-server1.log 2>&1 &`.
```

Comment:
We need to add steps to ensure the eBPF package repository service is running locally. ```cd /srv/l3afd
python3 -m http.server 8000 &```